### ISSUE TITLE
Reflect and `:Fnl` evaluation error on `//` operator.

### ISSUE THREAD

**Issue by monkoose** [rktjmp/hotpot.nvim#86]
Evaluating any code that includes some form of `(// x x)` with reflect api or `:Fnl` commands cause error `runtime error: hotpot-live-eval:2: unexpected symbol near '/'`

Seems like some escaping for `/` is required.

**Comment by rktjmp** [comment]
https://fennel-lang.org/reference#operators

> These all work as you would expect, with a few caveats. The bitwise operators are only available in Lua 5.3+, unless you use the --use-bit-lib flag or the useBitLib flag in the options table, which lets them be used in LuaJIT. **The integer division operator (//) is only available in Lua 5.3+.**

Neovim only embeds 5.1-jit so I don't believe this is technically valid fennel code in that runtime, it's present in Fennel upstream:

```
λ fennel --version
Fennel 1.2.0 on PUC Lua 5.4
λ fennel -e "(// 10 10)"
1
```

```
λ fennel --version
Fennel 1.2.0 on LuaJIT 2.1.0-beta3 Linux/x64
λ fennel -e "(// 10 10)"
runtime error: [string "(// 10 10)"]:1: unexpected symbol near '/'
stack traceback:
  [C]: in function 'assert'
  ...--3.4.0/lib/luarocks/rocks-5.1/fennel/1.2.0-1/bin/fennel:972: in function 'load-code'
  ...--3.4.0/lib/luarocks/rocks-5.1/fennel/1.2.0-1/bin/fennel:5328: in function ?
  [C]: in function 'xpcall'
  ...--3.4.0/lib/luarocks/rocks-5.1/fennel/1.2.0-1/bin/fennel:6040: in function 'dosafely'
  ...--3.4.0/lib/luarocks/rocks-5.1/fennel/1.2.0-1/bin/fennel:6255: in function ?
  [C]: in ?
```

**Comment by monkoose** [comment]
Ah my bad. Sorry for false report and thank you for explanation. I do tested it in command line fennel forgot that it uses lua 5.4 by default.
